Framework/Crystal/test/IndexPeaksWithSatellitesTest.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * IndexPeaksWithSatellitesTest.h
+ *
+ *  Created on: July, 2018
+ *      Author: Vickie Lynch
+ */
+
+#ifndef INDEXPEAKSWITHSATELLITESTEST_H_
+#define INDEXPEAKSWITHSATELLITESTEST_H_
+
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/FrameworkManager.h"
+#include "MantidCrystal/FindUBUsingIndexedPeaks.h"
+#include "MantidCrystal/IndexPeaksWithSatellites.h"
+#include "MantidCrystal/LoadIsawPeaks.h"
+#include "MantidCrystal/LoadIsawUB.h"
+#include "MantidDataObjects/PeaksWorkspace.h"
+#include "MantidKernel/System.h"
+#include "MantidKernel/Timer.h"
+#include <cxxtest/TestSuite.h>
+
+using namespace Mantid::Crystal;
+using namespace Mantid::API;
+using namespace Mantid::DataObjects;
+using namespace Mantid::Kernel;
+
+class IndexPeaksWithSatellitesTest : public CxxTest::TestSuite {
```

Comment:
I'm still not satisfied with the level of testing here. Have we really tested a decent subset of the edge cases? Some cases to think about:
 - Different numbers/ combinations of mod vectors?
 - With/without *RoundHKLs*
 - With/without *CommonUBForAll*
 - With/without *GetVectorsFromUB*
 - With/without *CrossTerms*
 - Testing the output properties actually return the correct values.

Please try to add simple test cases for each of these, and not test everything in one large method.